guides/v2.2/frontend-dev-guide/css-topics/css-preprocess.md

```diff
@@ -102,7 +102,7 @@ bin/magento setup:static-content:deploy
 The tool pre-processes (including compilation) and publishes the static view files.
 
 {:.bs-callout .bs-callout-info}
-Manual static content deployment is not required in "default" and "developer" modes. If you still want to deploy in these modes, use the -f option: `bin/magento setup:static-content:deploy -f`
+Manual static content deployment is not required in "default" and "developer" modes. If you still want to deploy in these modes, use the -f option: `bin/magento setup:static-content:deploy -f [<languages>]` (Default language is en_US).
```

Comment:
Maybe it's better to add the link to the document about the static content deployment instead of specifying only one parameter? The problem is after the proposed change it looks like the `[languages]` is the required option but it's not. Also, we have over 15 other optional parameters for the static content deploy that are not listed here. My suggestion is:
> Manual static content deployment is not required in "default" and "developer" modes. If you still want to deploy in these modes, use the -f option: `bin/magento setup:static-content:deploy -f`. Check more details about the command in the [Deploy static view files](https://magento-devdocs.github.io/devdocs-for-tests/guides/v2.1/config-guide/cli/config-cli-subcommands-static-view.html) section.

Reply:
@rogyar thanks for the suggestion,  please review